cdap-distributions/src/packer/cdap-sdk-ubuntu16-with-uri.json

```diff
@@ -172,11 +172,44 @@
     {
       "type": "chef-solo",
       "remote_cookbook_paths": "/var/chef/cookbooks",
-      "run_list": "recipe[openssh],recipe[cdap::sdk]",
+      "run_list": "recipe[openssh]",
+      "skip_install": true,
+      "json": {
+        "openssh": {
+          "server": {
+            "permit_root_login": "without-password",
+            "password_authentication": "no",
+            "use_dns": "no"
+          }
+        }
+      },
+      "except": ["cdap-cloud-sandbox-azure"]
+    },
+    {
+      "type": "chef-solo",
+      "remote_cookbook_paths": "/var/chef/cookbooks",
+      "run_list": "recipe[openssh]",
+      "skip_install": true,
+      "json": {
+        "openssh": {
+          "server": {
+            "client_alive_interval": "120",
```

Comment:
The AMI just requires that root can only login `without-password` and that DNS is disabled. Previously, we simply randomly scrambled the root password but otherwise left the configuration default. I think it's fine to add it globally and that's what I would do. Up to you, as this code is fine, either way.